dvc/command/plots.py

```diff
@@ -274,14 +337,17 @@ def _add_props_arguments(parser):
     )
 
 
-def _add_output_arguments(parser):
+def _add_output_argument(parser, typ="plots"):
     parser.add_argument(
         "-o",
         "--out",
         default=None,
-        help="Destination path to save plots to",
+        help=f"Destination path to save {typ} to",
```

Comment:
```suggestion
        help=f"Directory to save {typ} to.",
```
Since you mentioned it seems awkward that `-o` specifies a directory instead of the actual filepath, should we be more explicit about it in help?